File: solutions/golang/restaurantmanagementsystem/menu_item.go
package restaurantmanagementsystem

type MenuItem struct {
	ID          int
	Name        string
	Description string
	Price       float64
	Available   bool
}

func NewMenuItem(id int, name, description string, price float64, available bool) *MenuItem {
	return &MenuItem{
		ID:          id,
		Name:        name,
		Description: description,
		Price:       price,
		Available:   available,
	}
}

File: solutions/csharp/restaurantmanagementsystem/MenuItem.cs
namespace RestaurantManagementSystem
{
    public class MenuItem
    {
        public int Id { get; }
        public string Name { get; }
        public string Description { get; }
        public double Price { get; }
        public bool Available { get; }

        public MenuItem(int id, string name, string description, double price, bool available)
        {
            Id = id;
            Name = name;
            Description = description;
            Price = price;
            Available = available;
        }
    }
}